src/hooks/useConsoleErrors.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+import { ResultsPageOptions } from '../types';
+
+const useConsoleErrors = (quizId: string, resultsPageOptions: ResultsPageOptions) => {
+  useEffect(() => {
+    if (!quizId) {
+      // eslint-disable-next-line no-console
+      console.error('quizId is a required field of type string');
+    }
+
+    if (!resultsPageOptions || Object.keys(resultsPageOptions).length === 0) {
+      // eslint-disable-next-line no-console
+      console.error('resultsPageOptions is a required field of type object');
+    }
+
+    if (resultsPageOptions && !resultsPageOptions?.onAddToCartClick) {
```

Comment:
This is checking if we have `resultsPageOptions` is true to also check if it has `onAddToCartClick `

The above is checking if no `resultsPageOptions` just console error. If it's undefined no need to also check `onAddToCartClick ` since it will also be undefined